test/integration/release-integration.ts

```diff
@@ -216,5 +217,174 @@ describe('Release Api', async function () {
         })
       }
     })
+
+    test('publish Release', async () => {
+      // keeps original version
+      const entry = await testEnvironment.getEntry(TestDefaults.entry.testEntryReferenceId)
+
+      const release = await testEnvironment.createRelease({
+        title: 'Release (test)',
+        entities: {
+          sys: { type: 'Array' },
+          items: [makeLink('Entry', TestDefaults.entry.testEntryReferenceId)],
+        },
+      })
+
+      const publishAction = await release.publish()
+      const updatedEntry = await testEnvironment.getEntry(TestDefaults.entry.testEntryReferenceId)
+
+      expect(publishAction.sys.status).to.eql('succeeded')
+
+      // Publishing a Release publishes all the entities within, so an entry will have its version increased
+      expect(updatedEntry.sys.version).to.eql(entry.sys.version + 1)
+
+      // cleanup
+      await testEnvironment.deleteRelease(release.sys.id)
```

Comment:
Do we need to do this in a `finally` block?

Reply:
was this in the line 203? 🤔 

(L242 I'm not using `try`)